packages/core/src/utils/components.ts

```diff
@@ -1,17 +1,19 @@
 import { CONTENTFUL_COMPONENTS } from '../constants';
 
+const structureComponents = new Set([
+  CONTENTFUL_COMPONENTS.section.id,
+  CONTENTFUL_COMPONENTS.columns.id,
+  CONTENTFUL_COMPONENTS.container.id,
+  CONTENTFUL_COMPONENTS.singleColumn.id,
+]);
+
 export const isContentfulStructureComponent = (componentId?: string) =>
```

Comment:
What is the point of making this input param of the function optional? I feel like we should just require a componentId in this function.

Reply:
Basically there are several places in the UI and SDK where is function is called but the componentId might be undefined, so we simplify the implementation by allowing the param to be undefined as well.

![Screenshot 2024-02-29 at 2 24 55 PM](https://github.com/contentful/experience-builder/assets/8539634/0063a632-42a0-46e8-9f1d-6eebd357acad)